torch/csrc/jit/constants.cpp

```diff
@@ -129,8 +130,47 @@ RegisterOperators reg({
 });
 
 c10::optional<IValue> toIValue(const Value* v) {
-  if(v->node()->kind() != prim::Constant)
+  if (v->node()->kind() == prim::ListConstruct) {
```

Comment:
I don't like this additional logic here. It is essentially duplicating constant propagation.  If this is just needed to make the test harness work, can it just be put in that source file as a separate function? Everything that is constant should eventually become a prim::Constant in the

Reply:
sure, that makes sense